nixos/modules/services/misc/plikd.nix

```diff
@@ -0,0 +1,84 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let
+  cfg = config.services.plikd;
+
+  format = pkgs.formats.toml {};
+
+  plikdCfg = format.generate "plikd.cfg" cfg.settings;
+in
+{
+  options = {
+    services.plikd = {
+      enable = mkEnableOption "Enable the plikd server";
+
+      user = mkOption {
```

Comment:
As mentioned above, you can probably drop this option and use `DynamicUser` instead.